docs/azure_aks_scenario.md

```diff
@@ -0,0 +1,238 @@
+# Scenario: AKS cluster creation & attach
+
+This scenario will demonstrate the multi-cloud capabilities of compositions by leveraging ASO to manage Azure resources. We use the example of creating an Azure AKS cluster and attaching it to GKE Enterprise.
+
+## Deploy an attached AKS cluster with composition
+
+With composition, users can separate the infrastructure creation tasks to platform team and user teams. The platform team can define the composition with all the dependencies of the attached AKS cluster. On the other hand, the user team can provision the attached AKS cluster based on their needs with the composition defined by the platform team.
+
+Here is an example of an attached AKS cluster. (The complete sample is uploaded in [github](https://github.com/cloud-native-compositions/compositions/samples/AttachedAKS). We will go through it in this guide.)
+
+### Platform Team
+
+#### **Step 1: Create the composition for the attached AKS cluster**
```

Comment:
The Azure docs are too GCP focused IMO, but that's OK for now we can improve later.

Reply:
Same for the AWS docs

Reply:
I was hesitant to add these here. But the alternative was no azure/aws examples at all. Agreed we need to come up with installation on Azure/AWS and then use examples that dont need GCP bits.